sz/Traffic/src/traffic/relogo/PoissonStream.groovy

```diff
@@ -12,7 +12,7 @@ import repast.simphony.relogo.schedule.Setup
 import traffic.ReLogoTurtle
 
 class PoissonStream extends ReLogoTurtle {
-	def rate = 0.2
+	def poissonStreamRate = 0.2
```

Comment:
In order to use global variable from the slider, shouldn't you get rid of this property?

Reply:
True, fixing